src/v0/destinations/am/transform.js

```diff
@@ -499,6 +499,14 @@ function responseBuilderSimple(
           delete payload.user_properties.address.country;
         }
       }
+
+      if (!payload.user_id && !payload.device_id) {
+        logger.debug("Either of user ID or device ID fields must be specified");
```

Comment:
[nitpick] Would a debug-log add value here ?

Reply:
This is a non-blocking comment

Reply:
Followed the existing convention in the amplitude destination, but later realised it's outdated. I will once go through the amplitude doc and accordingly will try to raise a refactor/ new V1 version transformer task, where I'll try to cover all things. For now will merge this